databricks/koalas/frame.py

```diff
@@ -2191,6 +2191,111 @@ def to_parquet(self, path: str, mode: str = 'error',
         self._sdf.write.parquet(path=path, mode=mode, partitionBy=partition_cols,
                                 compression=compression)
 
+    def to_csv(self, path, mode=None, compression=None, sep=None, quote=None, escape=None,
+               header=None, nullValue=None, escapeQuotes=None, quoteAll=None, dateFormat=None,
+               timestampFormat=None, ignoreLeadingWhiteSpace=None, ignoreTrailingWhiteSpace=None,
+               charToEscapeQuoteEscaping=None, encoding=None, emptyValue=None):
+        """
+        Saves the content of the :class:`DataFrame` in CSV format at the specified path.
+
+        Parameters
+        ----------
+        path: str, required
+            Path to write to.
+        mode: str {'append', 'overwrite', 'ignore', 'error', 'errorifexists'}, default 'error'.
+            Specifies the behavior of the save operation when data already exists.
+
+            - ``append``: Append contents of this :class:`DataFrame` to existing data.
+            - ``overwrite``: Overwrite existing data.
+            - ``ignore``: Silently ignore this operation if data already exists.
+            - ``error`` or ``errorifexists`` (default case): Throw an exception if data already
+                exists.
+        compression: str {'none', 'uncompressed', 'snappy', 'gzip', 'lzo', 'brotli', 'lz4', 'zstd'}
+            Compression codec to use when saving to file. This can be one of the known
+            case-insensitive shorten names (none, bzip2, gzip, lz4, snappy and deflate).
+        sep: str, default None
+            Sets a single character as a separator for each field and value. If None is set,
+            it uses the default value, ``,``.
+        quote: str, default None
+            Sets a single character used for escaping quoted values where the separator can be part
+            of the value. If None is set, it uses the default value, ``"``.
+            If an empty string is set, it uses ``u0000`` (null character).
+        escape: str, default None
+            Sets a single character used for escaping quotes inside an already quoted value.
+            If None is set, it uses the default value, ``\``.
```

Comment:
@RainFung 
This ``\`` is the root of the problem. 
Pydocstyle checks passed when I add `r` at the beginning of the docstring. 
```python 
r"""
Saves the content of the ...
```

Please make sure that the documentation still rendering correctly.